### ISSUE TITLE
Update Documentation to Modify Supported PHP Version to 5.4<=

### ISSUE THREAD

**Issue by magneticstain** [magneticstain/PrivyPaste#37]
Currently, PrivyPaste is listed as working on all versions of PHP 5.3<=. PHP 5.3 does not support usage of brackets, so this line will cause an HTTP 500 error to generate:

From `/lib/databaser.php` on line 324:

```
 public function queryDb($sql, $action = 'select', $sqlParams = [])
 {
          <truncated>
 }
```

The documentation should be updated to reflect the proper supported PHP version.

**Comment by magneticstain** [comment]
Alternatively, we could update the function parameters to be `$sqlParams = ''), then set it to a blank array if blank.

**Comment by magneticstain** [comment]
Two other PHP functions - `hex2bin()` and `mb_check_encoding()` - are also not supported in <5.4. Sounds like it might be easier to just drop support for <5.4.

**Comment by magneticstain** [comment]
Fixed with commit da3b178.
